etc/ansible/roles/network-runner/providers/dellemc.os9.os9/conf_access_port.yaml

```diff
@@ -9,13 +9,60 @@
   ansible.builtin.set_fact:
     port_description: "{{ port_description_cmd.stdout[0] | regex_search('(?<=description ).+?(?=[\\n\\r])') }}"
 
+- name: "dellemc.os9: get existing vlans from running config"
+  dellemc.os9.os9_command:
+    commands: "show interfaces switchport {{ port_name }}"
+  register: port_vlans_cmd
+  connection: network_cli
```

Comment:
What does the output of the `show interfaces switchport <port>` command look like?

Reply:
```

Codes:  U - Untagged, T - Tagged
        x - Dot1x untagged, X - Dot1x tagged
        G - GVRP tagged, M - Trunk
        i - Internal untagged, I - Internal tagged, v - VLT untagged, V - VLT tagged

Name: TenGigabitEthernet 1/25/1
802.1QTagged: Hybrid
Vlan membership:
Q       Vlans
U       623
T       498,500-502

Native VlanId:    623.


```